App/pom.xml

```diff
@@ -68,11 +67,11 @@
                         <list>com.gluonhq.chat.model.ChatMessage</list>
                         <list>com.gluonhq.chat.model.ChatImage</list>
                         <list>org.glassfish.json.JsonProviderImpl</list>
+                        <!-- for desktop only -->
+<!--                        <list>com.gluonhq.attach.orientation.impl.DesktopOrientationService</list>-->
+<!--                        <list>com.gluonhq.attach.position.impl.DesktopPositionService</list>-->
```

Comment:
Can we move client configuration to profile?

Reply:
Yes, that is an option here. My preference is not to add the plugins to the profiles, as you end up duplicating a lot of code.

An alternative in this case is to transform lists into comma separated strings in the client plugin, so they can be set as properties. Then you will define the reflectionList as a property that can be set in each profile.

Another alternative is in Attach, as it should allow mocking the desktop services more easily...